packages/apply/src/index.ts

```diff
@@ -267,8 +267,9 @@ const bumpAll = ({
     if (logs && !previewVersion)
       console.log(`bumping ${pkg} with ${changes[pkg].type}`);
     if (previewVersion)
+      // change log (assume that the prerelease will be removed)
       console.log(
-        `bumping ${pkg} to ${packageFiles[pkg].version}-${previewVersion} to publish a preview`
+        `bumping ${pkg} with ${previewVersion} template to publish a preview`
```

Comment:
I wonder if `identifier` might be better used instead of `template`?

Reply:
Done 👍